kafka/README.md

```diff
@@ -55,7 +76,10 @@ channels:
           clientId:
             type: string
             enum: ['myClientId']
-          bindingVersion: '0.1.0'
+          schemaIdLocation: 'payload'
+          schemaIdPayloadEncoding: 'apicurio-new'
+          schemaLookupStrategy: 'TopicIdStrategy'
```

Comment:
Thinking about it again, I wonder if those properties are really tied to operation and shouldn't be bound to channel instead. 🤔  I've added them to operation because there already was binding information at that level and adding a new binding at the channel level may certainly make things complex... However, I can't think of a situation where those could be different depending on the operation.
@dalelane Would you have an opinion on that?

Reply:
On the same idea, I think it would be nice to add another PR to allow the specification of the topic name as well as the number of configured partitions. IMHO, theses information would strengthen the need for channel-level bindings. Thus we could move there the `schema*` properties.

Any thoughts or opinions?

Reply:
> I wonder if those properties are really tied to operation and shouldn't be bound to channel instead

yeah, I agree - I think that makes more sense

Reply:
> I think it would be nice to add another PR to allow the specification of the topic name as well as the number of configured partitions

There was some discussion of that in https://github.com/asyncapi/spec/issues/623 - I think it'd be really useful, as it's a good pointer to tell you how many clients you may want to run.

Reply:
Thinking about it once again ... I think it is even more correct to move schema encoding-related info to the message level. Typically - end even though I don't personally consider this as a best practice - you could have different kinds of Avro records on the same topic. Apicurio also proposes a per-record strategy so I think moving this to the message level will increase the capabilities of describing all kinds of configurations.

Reply:
Just pushed a new commit with that change + new channel level information for:
* `topic` in case it's different from the channel name,
* `partitions` that represents the number of configured partitions,
* `replicas` that represents the number of configured replicas for this topic.